sgtm/apollo/gtm/variable_template.tpl

```diff
@@ -0,0 +1,257 @@
+﻿___INFO___
+
+{
+  "type": "MACRO",
+  "id": "cvt_temp_public_id",
+  "version": 1,
+  "securityGroups": [],
+  "displayName": "gps-apollo",
+  "description": "Get data from a Google Sheet in realtime. A quick \u0026 lightweight route to get data without needing more complex cloud setups.",
+  "containerContexts": [
+    "SERVER"
+  ]
+}
+
+
+___TEMPLATE_PARAMETERS___
+
+[
+  {
+    "type": "TEXT",
+    "name": "spreadsheet_id",
+    "displayName": "The spreadsheet id that we get the data from.",
+    "simpleValueType": true
+  },
+  {
+    "type": "TEXT",
+    "name": "age",
+    "displayName": "The value of age group that we input in the form.",
+    "simpleValueType": true
+  },
+  {
+    "type": "TEXT",
+    "name": "address",
+    "displayName": "The address that we input in the form.",
+    "simpleValueType": true
+  },
+  {
+    "type": "TEXT",
+    "name": "job",
+    "displayName": "The related job that we input in the form.",
+    "simpleValueType": true
+  }
+]
+
+
+___SANDBOXED_JS_FOR_SERVER___
+
+// Enter your template code here.
+const getGoogleAuth = require("getGoogleAuth");
```

Comment:
nit: please use consistent quotes, so either single or double. It looks like single quotes `'` are used more, so perhaps update these? Thanks

Reply:
Thank you so much for checking and your opinion.
I was supposed to use single quotes, but I went for the double quotes because I would like to be consistent between other our solution codes and this solution code. I think you preferred double quotes rather than sing quotes and there are also double quotes in Phoebe, so I decided to use double quotes. But if my understanding is incorrect, please let me know.

Thank you!!